### DISCUSSION TITLE
2024-08 topics to discuss at the next sync

### DISCUSSION THREAD

**Discussion by WitchDevelops** [WitchDevelops/cardboard-fort#63] [Q&A]
1. error component for database call errors - design
2. date picker component - when navigating back several years, there's a lot of clicking
3. delete pet - wording, modal design
4. top nav / side nav - which one we want, what to put where, mobile vs desktop design
5. unit tests - i hate them and they slow you down, but maybe? or end-to-end tests? or both? I think soon we should start adding tests
6. state management - useContext vs something else?
7. go through `Issues` and discuss them

**Comment by jagodakubicka** [comment]
1. Figma / Jagoda
2. find replacement / Domi
3. for now 'delete' for button and 'delete pet data' in modal - will change later for sth nicer
4. figma / Jagoda
5. Cypress for start / Domi
6. useContext / Jagoda
